### DISCUSSION TITLE
Navigation menu translation

### DISCUSSION THREAD

**Discussion by stonebuzz** [squidfunk/mkdocs-material#5653] [Q&A] [answered]
Hello everyone, 
First of all, thank you for this powerful tool that can be configured as you wish =)

I'm looking to translate the navigation menu (in ```french```).

![image](https://github.com/squidfunk/mkdocs-material/assets/7335054/de024a30-b65f-40f2-8468-27cc95803295)

Generated menu depends on the organization of the folders located in ```docs```.

```
├── administration
│   ├── process
│   │   ├── process_article1.fr.md
│   │   ├── process_article1.md
│   │   ├── process_article2.fr.md
│   │   ├── process_article2.md
│   └── rules
│       ├──  rule_article1.fr.md
│       ├──  rule_article1.md
│       ├──  rule_article2.fr.md
│       └──  rule_article2.md
├── assets
│   ├── assets_article1.fr.md
│   ├── assets_article2.md
```

It doesn't seem possible to translate the navigation menu (depending of folder name).

It is only possible to redefine existing translations.

https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#custom-translations

has anyone tried / succeeded in translating the navigation menu?

Perhaps via a twig macro?

Thank you for your help

**Comment by squidfunk** [comment, marked as answer]
This is something that should be raised to the maintainers of the [mkdocs-static-i18n](https://github.com/ultrabug/mkdocs-static-i18n) plugin, which you seem to be using as can be inferred from the file endings. We don't officially support that plugin, but there may be a way to do it 😊

**Comment by hendrikp** [comment]
I recommend to split the languages to seperate sites not using file endings in same dir, and use literate nav to setup navigation for each language indepently.

This way the search and rest of site can be seperated and generated correctly per language.

Works great but might require some hardlinks/copying for shared resources.
